basics/generate_pdf.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require 'pdf.php';
+
+/**
+ * generates pdf using user input data
+ *
+ * @param $full_name
+ *   stores full name
```

Comment:
It should be like
```
 * @param string $full_name
 *   User's full name.
```

1/ Add proper descriptions.
2/ Always mention data type of params.